conf/riff-raff.yaml

```diff
@@ -0,0 +1,21 @@
+stacks: [membership]
+regions: [eu-west-1]
+deployments:
+  cfn:
+    type: cloud-formation
+    app: contributions-frontend
+    parameters:
+      prependStackToCloudFormationStackName: false
```

Comment:
Might be worth trying to switch to `cloudFormationStackByTags` from https://riffraff.gutools.co.uk/docs/magenta-lib/types#cloudformation which is the default for riff-raff.yml, then we don't need the prepend nonsense.